src/SourceBuild/content/eng/tools/CreateBaselineUpdatePR/Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<Project>
+
+  <!--
+    Do not import the Arcade SDK as this would introduce a prebuilt as this project builds
+    before the custom sdk resolver is available in source-only mode.
+  -->
```

Comment:
Does this library build as part of the VMR orchestrator build? If not then I would remove this comment and file completely. I think it should be fine to use the Arcade SDK here.

Reply:
Good point, this shouldn't get built as part of the VMR. I'll remove this file.